docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
+> * Tasks matching keywords will be returned
+
+Examples: 
+* `search oct 1`<br>
+  Lists all tasks that have the date `oct 1`.
+* `search milk`<br>
+  Lists the all the tasks with names containing the word `milk` if any can be found.
+  
 #### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+Deletes the specified task from the list. Irreversible.<br>
+Format: `delete INDEX...`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
+* `delete 2`<br>
+  Deletes the task indexed number second in the current seen list.
+* `search buy milk`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the task indexed first in the results of the `search` command.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+To delete more than one task, simply add in the other indexes separated with a comma. <br>
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
-  The index refers to the index number shown in the most recent listing.<br>
-  The index **must be a positive integer** 1, 2, 3, ...
+Example:<br>
+* `delete 1, 2, 3`
+* `search oct 1`
+  `done 2, 3`
+  
+  
+#### Undo : `undo`
+Reverts the most recent action.<br>
+Format: `undo`
+
+#### Editing a task : `edit`
+Edits task on the displayed list.<br>
+Format: `edit INDEX PARAMETER NEW_INFORMATION`
+
+<img src="images/2ndimpt.png" width="500"><br>
 
 Examples: 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br> 
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `edit 1 date oct 11`
+* `list dec 30`<br>
+  `edit 5 task buy coconut`
+  
+To make one of the parameters (except the task name) empty, follow the format below:<br>
+Format: `edit INDEX PARAMETER clear`
+
+Example: 
+* `edit 3 time clear`
+
+
+#### Sorting all tasks : `sort`
+Sort all tasks according to the specified category.<br>
+Three categories: Alphabetical order, priority level, deadlines<br>
+Format: `sort CATEGORY`
+
+<img src="images/sortpic.png" width="400"><br>
+
+Example:<br>
+* `sort A`
+* `sort D`
+
+#### Reserving a time slot : `reserve`
+Reserves a time slot and prevents you from scheduling conflicting tasks.<br>
+Format: `reserve DATE, TIME`
+
+Example:<br>
+* `reserve mon, 1200-2300`
+* `reserve oct 15, 1500-1500`
+
+#### Marking a task as complete : `done`
+Marks task on displayed list as complete. Does not remove it from the list. The completed tasks will be seen striked off.<br>
```

Comment:
it may be prudent to remove the marked task from the list displayed and maintain it in a separate list. you can have a command to list all completed tasks. again imagine a UI where, out of the 100 tasks, top 80 are completed. to see the incomplete tasks i have to scroll all the way down in the UI.